app/components/container/container.html

```diff
@@ -304,3 +304,33 @@
     </rd-widget>
   </div>
 </div>
+
+<div class="row" ng-if="!(availableNetworks | emptyobject)">
```

Comment:
Instead of creating a new panel, could you try to insert this feature inside the existing Connected network panel? Below the network table would be fine I think.

![portainer](https://user-images.githubusercontent.com/5485061/27253303-22cc1b6a-5372-11e7-95a4-31a78a5d90f3.png)